functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -505,11 +501,7 @@ function New-DbaAvailabilityGroup {
             # something is up with .net create(), force a stop
             Invoke-Create -Object $ag
         } catch {
-            $msg = $_.Exception.InnerException.InnerException.Message
-            if (-not $msg) {
-                $msg = $_
-            }
-            Stop-Function -Message $msg -ErrorRecord $_ -Target $Primary
+            Stop-Function -ErrorRecord $_ -Target $Primary
```

Comment:
This removes the entire message, unfortunately. Sometimes the error record isn't as detailed, so I do the cheesy way around with `$msg = $_.Exception.InnerException.InnerException.Message`, which is why you see this here. PS and SMO aren't perfect so sometimes you have to put in workarounds.